### ISSUE TITLE
The Loss Problems

### ISSUE THREAD

**Issue by zhenglm** [YerevaNN/R-NET-in-Keras#19]

**Comment by zhenglm** [comment]
Hello, I'm training the R-Net with Python3 as your instructions , but I get a loss value about 2.3842e-07 after only one epoch. And the acc is still low (val_answer_start_acc: 0.0247 - val_answer_end_acc: 0.0058). Is there something wrong with my training. Can you give some advices please?
